### PR TITLE
(SIMP-2232) Create cron module

### PR DESCRIPTION
**Pull request by jeefberkey**
Material used to be in simplib.

SIMP-2232 #close
SIMP-1679 #comment created cron module

### CODE REVIEW

File: data/common.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
```
Comment (jeefberkey) [inline data/common.yaml]: This is unnecessary, put defaults in the manifests

File: metadata.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      "operatingsystem": "RedHat",
+      "operatingsystemrelease": [
+        "6",
+        "7"
+      ]
+    }
+  ],
+    "requirements": [
+    {
+      "name": "pe",
```
Comment (jeefberkey) [inline metadata.json]: Make this puppet and pe, from skeleton

File: spec/acceptance/nodesets/centos-6-x64.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+HOSTS:
```
Comment (jeefberkey) [inline spec/acceptance/nodesets/centos-6-x64.yaml]: Remove unneeded nodesets

File: README.md [hunk 4]
```diff
@@ -1,2 +1,86 @@
...
+3. [Usage - Configuration options and additional functionality](#usage)
+4. [Reference - An under-the-hood peek at what the module is doing and how](#reference)
+5. [Limitations - OS compatibility, etc.](#limitations)
+6. [Development - Guide for contributing to the module](#development)
+    * [Acceptance Tests - Beaker env variables](#acceptance-tests)
+
+
+## Description
+
+This module manages the cron packges, service, and /etc/cron.allow.
```
Comment (lnemsick-simp) [inline README.md]: 'cron packages' not 'cron packges'

File: README.md [hunk 5]
```diff
@@ -1,2 +1,86 @@
...
+
+### This is a SIMP module
+This module is a component of the [System Integrity Management Platform](https://github.com/NationalSecurityAgency/SIMP), a compliance-management framework built on Puppet.
+
+If you find any issues, they can be submitted to our [JIRA](https://simp-project.atlassian.net/).
+
+This module is optimally designed for use within a larger SIMP ecosystem, but it can be used independently:
+
+ * When included within the SIMP ecosystem, security compliance settings will be managed from the Puppet server.
+ * If used independently, all SIMP-managed security subsystems are disabled by default and must be explicitly opted into by administrators.  Please review the `$client_nets`, `$enable_*` and `$use_*` parameters in `manifests/init.pp` for details.
```
Comment (lnemsick-simp) [inline README.md]: client_nets, enable_, and use_ are no longer relevant

File: README.md [hunk 6]
```diff
@@ -1,2 +1,86 @@
...
+
+SIMP Puppet modules are generally intended for use on Red Hat Enterprise Linux and compatible distributions, such as CentOS. Please see the [`metadata.json` file](./metadata.json) for the most up-to-date list of supported operating systems, Puppet versions, and module dependencies.
+
+
+## Development
+
+Please read our [Contribution Guide] (http://simp-doc.readthedocs.io/en/stable/contributors_guide/index.html).
+
+
+### Acceptance tests
```
Comment (lnemsick-simp) [inline README.md]: This module has not acceptance tests.  Perhaps this section should be removed or replace with spec test instructions.

**Pull request merged**